lib/modules/datasource/sbt-package/index.ts

```diff
@@ -22,7 +26,7 @@ export class SbtPackageDatasource extends MavenDatasource {
 
   override readonly defaultVersioning = ivyVersioning.id;
 
-  override readonly registryStrategy = 'hunt';
+  override readonly registryStrategy: RegistryStrategy = 'hunt';
```

Comment:
Now we've changed `sbt-plugin` to use the `merge` strategy - should `sbt-package` still be using `hunt` or should we change it to use `merge` too?

Reply:
No, let's keep it as is, it should be fine